docs/zh_cn/reference/command_reference.md

```diff
@@ -115,786 +113,398 @@ sudo cp hack/autocomplete/bash_autocomplete /etc/bash_completion.d/juicefs
+|`ADDRESS`|WebDAV 服务监听的地址与端口，例如：`localhost:9007`|
+|`--cert-file`|HTTPS 证书文件|
+|`--key-file`|HTTPS 密钥文件|
+|`--gzip`|通过 gzip 压缩提供的文件（默认值：false）|
+|`--disallowList`|禁止列出目录（默认值：false）|
+|`--access-log value`|访问日志的路径|
 
-`--limit value`<br />
-限制将要处理的对象的数量 (默认：-1)
+## `juicefs sync`
 
-`--manager value`<br />
-管理者地址
+在两个存储之间同步数据，阅读[「数据同步」](../guide/sync.md)以了解更多。
 
-`--worker value`<br />
-工作节点列表 (使用逗号分隔)
-
-`--bwlimit value`<br />
-限制最大带宽；单位为 Mbps (0 表示不限制) (默认：0)
-
-`--no-https`<br />
-不要使用 HTTPS (默认：false)
-
-`--check-all`<br />
-验证源路径和目标路径中所有文件的数据完整性 (默认：false)
-
-`--check-new`<br />
-验证新拷贝文件的数据完整性 (默认：false)
-
-#### 示例
+### 概览
 
 ```shell
+juicefs sync [command options] SRC DST
+
 # 从 OSS 同步到 S3
-$ juicefs sync oss://mybucket.oss-cn-shanghai.aliyuncs.com s3://mybucket.s3.us-east-2.amazonaws.com
+juicefs sync oss://mybucket.oss-cn-shanghai.aliyuncs.com s3://mybucket.s3.us-east-2.amazonaws.com
 
-# 从 S3 同步到 JuiceFS
-$ juicefs mount -d redis://localhost /mnt/jfs
-$ juicefs sync s3://mybucket.s3.us-east-2.amazonaws.com/ /mnt/jfs/
+# 从 S3 直接同步到 JuiceFS
+juicefs sync s3://mybucket.s3.us-east-2.amazonaws.com/ jfs://VOL_NAME/
 
 # 源端: a1/b1,a2/b2,aaa/b1   目标端: empty   同步结果: aaa/b1
-$ juicefs sync --exclude='a?/b*' s3://mybucket.s3.us-east-2.amazonaws.com/ /mnt/jfs/
+juicefs sync --exclude='a?/b*' s3://mybucket.s3.us-east-2.amazonaws.com/ jfs://VOL_NAME/
 
 # 源端: a1/b1,a2/b2,aaa/b1   目标端: empty   同步结果: a1/b1,aaa/b1
-$ juicefs sync --include='a1/b1' --exclude='a[1-9]/b*' s3://mybucket.s3.us-east-2.amazonaws.com/ /mnt/jfs/
+juicefs sync --include='a1/b1' --exclude='a[1-9]/b*' s3://mybucket.s3.us-east-2.amazonaws.com/ jfs://VOL_NAME/
 
 # 源端: a1/b1,a2/b2,aaa/b1,b1,b2  目标端: empty   同步结果: a1/b1,b2
-$ juicefs sync --include='a1/b1' --exclude='a*' --include='b2' --exclude='b?' s3://mybucket.s3.us-east-2.amazonaws.com/ /mnt/jfs/
+juicefs sync --include='a1/b1' --exclude='a*' --include='b2' --exclude='b?' s3://mybucket.s3.us-east-2.amazonaws.com/ jfs://VOL_NAME/
 ```
 
-### `juicefs rmr`
+源路径（`SRC`）和目标路径（`DST`）的格式均为 `[NAME://][ACCESS_KEY:SECRET_KEY[:SESSIONTOKEN]@]BUCKET[.ENDPOINT][/PREFIX]`，其中：
+
+- `NAME`：JuiceFS 支持的数据存储类型（如 `s3`、`oss`），请参考[文档](../guide/how_to_set_up_object_storage.md#supported-object-storage)。
+- `ACCESS_KEY` 和 `SECRET_KEY`：访问数据存储所需的密钥信息，请参考[文档](../guide/how_to_set_up_object_storage.md#aksk)。
+- `TOKEN` 用来访问对象存储的 token，部分对象存储支持使用临时的 token 以获得有限时间的权限
+- `BUCKET[.ENDPOINT]`：数据存储服务的访问地址，不同存储类型格式可能不同，具体请参考[文档](../guide/how_to_set_up_object_storage.md#supported-object-storage)。
+- `[/PREFIX]`：可选，源路径和目标路径的前缀，可用于限定只同步某些路径中的数据。
+
+### 参数
+
+|项 | 说明|
+|-|-|
+|`--start=KEY, -s KEY, --end=KEY, -e KEY`|提供 KEY 范围，来指定对象存储的 List 范围。|
+|`--threads=10, -p 10`|并发线程数，默认为 10。|
+|`--list-threads=1`|并发 `list` 线程数，默认为 1。阅读[并发 `list`](../guide/sync.md#concurrent-list)以了解如何使用。|
+|`--list-depth=1`|并发 `list` 目录深度，默认为 1。阅读[并发 `list`](../guide/sync.md#concurrent-list)以了解如何使用。|
```

Comment:
The link is broken

Reply:
will resolve after https://github.com/juicedata/juicefs/pull/3875